src/test/java/lambda/part1/exercise/Exercise1.java

```diff
@@ -0,0 +1,114 @@
+package lambda.part1.exercise;
+
+import com.google.common.base.Predicate;
+import com.google.common.collect.FluentIterable;
+import java.util.Comparator;
+import lambda.data.Person;
+import java.util.Arrays;
+import java.util.List;
+import org.junit.jupiter.api.Test;
+import static org.junit.jupiter.api.Assertions.assertArrayEquals;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+public class Exercise1 {
+
+    @Test
+    public void sortPersonsByAgeUsingArraysSortComparator() {
+        Person[] persons = getPersons();
+
+        // TODO использовать Arrays.sort
+
+        Comparator<Person> comparatorPersonByAdge = new Comparator<Person>() {
```

Comment:
You should add new class declaration which will implement Comrapator<Person> and use it's instance

Reply:
Ok. will change it in 10 minutes.